tests/components/rova/test_config_flow.py

```diff
@@ -0,0 +1,212 @@
+"""Tests for the Rova config flow."""
+from unittest.mock import Mock, patch
+
+import pytest
+from requests.exceptions import ConnectTimeout, HTTPError
+
+from homeassistant import data_entry_flow
+from homeassistant.components.rova.const import (
+    CONF_HOUSE_NUMBER,
+    CONF_HOUSE_NUMBER_SUFFIX,
+    CONF_ZIP_CODE,
+    DOMAIN,
+)
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_USER
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+ZIP_CODE = "7991AD"
+HOUSE_NUMBER = "10"
+HOUSE_NUMBER_SUFFIX = "a"
+
+
+@pytest.fixture(name="test_api")
+def mock_controller():
+    """Mock a successful Rova API."""
+    api = Mock()
+    api.is_rova_area.return_value = True
+
+    with patch("rova.rova.Rova", return_value=api):
```

Comment:
Please patch at the place where this is used. Checkout the `epion` tests for a nice way to patch the lib.